docs/Installation.md

```diff
@@ -1,58 +1,135 @@
+# Installation
+
 T4 has two different components:
-* A Python package, `t4`, distributed via `pip`
-* A web interface, the T4 Catalog, deployed using [AWS CloudFormation](https://aws.amazon.com/cloudformation/)
+* A Python package, `t4`
+* A web interface, the T4 Catalog
 
 If you have an already-provisioned catalog, only the Python package is needed.
 
+## T4 client (Python 3.6)
 
-## Installing the Python package
+```bash
+$ pip install t4
+```
 
-T4 is currently in pre-release alpha. To install the current T4 `master`, first ensure that you are in a Python 3.6 environment, then run:
+If you wish to use AWS resources, such as S3 buckets, you will need valid AWS credentials. If this is your first time using the AWS CLI, run the following:
 
 ```bash
-$ pip install git+https://github.com/quiltdata/t4.git#subdirectory=api/python
+$ pip install aws-cli
+$ aws configure
 ```
 
-In order to interface with a remote catalog you will also need valid AWS credentials. If this is your first time using AWS, ensure that you have the AWS CLI installed (`pip install aws-cli`), then run the following to store the IAM credentials you wish to use with T4:
+If you are already using the AWS CLI, you may use your existing profile, or [create a new profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html).
+
+### Developer
+
+Install the current T4 client from `master`:
 
 ```bash
-$ aws configure
+$ pip install git+https://github.com/quiltdata/t4.git#subdirectory=api/python
 ```
 
-If you are already using AWS IAM you may reuse your credentials for T4. Alternatively you can create a Quilt-specific [profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html).
+## Catalog
 
+The Javascript catalog is a single-page, static application.
+You can build it as follows:
 
-## Deploying the catalog
-The following instructions use CloudFormation to install T4 on a bucket in your AWS account.
+```
+$ cd catalog
+$ npm run build
+```
 
-* Log in to your AWS console
+The static assets, including webpack chunks, are written to `/static`.
 
-* Go to Services > CloudFormation > Create stack
+For more advanced deployment scripts, see [`deploy_navigator.sh`](../deployment/deploy_navigator.sh).
 
-![](imgs/start.png)
 
-* Click "Upload a template to Amazon S3" and select `t4.yaml`, provided to you by Quilt
-* Click Next
-* Fill in Stack name and Parameters
+## AWS T4 Instance (via CloudFormation)
+The following section uses CloudFormation to manage S3 buckets, ElasticSearch endpoints, and CloudFront to deploy
```

Comment:
You need to explain what these components are, e.g. state that these instructions are for deploying a catalog instance on an AWS account.

Reply:
Rather than that, I promised less.